include/jemalloc/internal/rb.h

```diff
@@ -560,6 +560,19 @@ a_prefix##reverse_iter_filtered(a_rbt_type *rbtree, a_type *start,	\
  * the same as with the unfiltered version, with the added constraint that the
  * returned node must pass the filter.
  */
+JEMALLOC_ALWAYS_INLINE void
+rb_remove_safety_checks(const void *nodep, const char *function_name) {
+	if (!config_opt_safety_checks) {
+		return;
+	}
+	if (unlikely(nodep == NULL)) {
+		safety_check_fail("<jemalloc>: Invalid deallocation detected in %s: "
```

Comment:
nit: over 80-char length too (In case you need to change your editor settings, each tab is 8 space wide).

Reply:
Fixed